fineract-client/build.gradle

```diff
@@ -56,6 +56,7 @@ task buildJavaSdk(type: org.openapitools.generator.gradle.plugin.tasks.GenerateT
     ]
     generateModelTests = false
     generateApiTests = false
+    templateDir = "$projectDir/config/swagger/templates"
```

Comment:
as above, let's see what we agree. Sorry I'm so adamant about not wanting to keep this simple.

Reply:
Yup, see my comments

Reply:
FYI: removed all template related references (also from client.md).